image_writers_guide/sti.adoc

```diff
@@ -94,12 +76,15 @@ make all
 make install
 popd
 ----
+====
 
-=== run
+[[run]]
+*`run`*
 
 The `run` script is responsible for executing your application.
 
```

Comment:
It's different statement, it's more like a v2 kind of application. That's true it's not applicable anymore to v3, but since we're trying to mimic v2 behavior with STI we need to come up with some naming here.